packages/multichain-network-controller/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { NetworkStatus } from '@metamask/network-controller';
+
+import type {
+  MultichainNetworkConfiguration,
+  MultichainNetworkMetadata,
+} from './MultichainNetworkController';
+
+export const bitcoinCaip2ChainId = 'bip122:000000000019d6689c085ae165831e93';
+export const solanaCaip2ChainId = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp';
+
+export const multichainNetworkConfigurations: Record<string, MultichainNetworkConfiguration> = {
+  bitcoinCaip2ChainId: {
+    chainId: bitcoinCaip2ChainId,
```

Comment:
Could be a `BtcScope` for now if we decide to use the `keyring-api` but again... IDK about depending on this package in this context 😅